README.md

```diff
@@ -191,18 +191,23 @@ $ npm install --save drivelist
 Documentation
 -------------
 
+
+* [drivelist](#module_drivelist)
+    * [.list(callback)](#module_drivelist.list)
+    * [.get(device, callback)](#module_drivelist.get)
+
 <a name="module_drivelist.list"></a>
 
 ### drivelist.list(callback)
-**Kind**: static method of <code>[drivelist](#module_drivelist)</code>
-**Summary**: List available drives
-**Access:** public
+**Kind**: static method of <code>[drivelist](#module_drivelist)</code>  
+**Summary**: List available drives  
+**Access:** public  
```

Comment:
The two spaces force a line break, so that those don't get squished into the same line.

> When you do want to insert a \<br /\> break tag using Markdown, you end a line with two or more spaces, then type return

http://daringfireball.net/projects/markdown/syntax#p